cmd/run.go

```diff
@@ -226,6 +226,11 @@ func SetupRunFlags(cmd *cobra.Command) {
 		"from an empty string to a non-empty string (or from a non-empty string to the empty string) requires a resync.")
 	cmd.PersistentFlags().Uint("state-syncer-mempool-txn-sync-limit", 10000, "The maximum number of transactions to "+
 		"process in the mempool tx state syncer at a time.")
+
+	// PoS Checkpoint Syncing
+	// TODO: a default value is a little tricky here since we don't know what network we're using.
```

Comment:
I'd get rid of the TODO and just make the comment explain that we actually DO set a default value, but in code rather than explicitly. I would also embed the default in the message for the flag. Like this:

```
fmt.Sprintf("A comma-separated list of URLs that ... . If this flag is unset it will default to %v for mainnet and %v for testnet", DefaultMainnetBlah, DefaultTestnetBlah)
```

Reply:
Hold on I'm going to see if I can edit this myself the way you did on my other PR...